spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingProperties.java

```diff
@@ -33,7 +35,17 @@
 @JsonInclude(Include.NON_DEFAULT)
 public class ChannelBindingProperties {
 
-	public static final String PATH = "path";
+	public static final String DESTINATION = "destination";
+
+	public static final String PARTITION_COUNT = "partitionCount";
+
+	public static final String PARTITIONED = "partitioned";
+
+	@Value("${INSTANCE_INDEX:${CF_INSTANCE_INDEX:0}}")
+	private int instanceIndex;
+
+	@Value("${INSTANCE_COUNT:1}")
```

Comment:
Yeah - the initial reasoning was that we could set it in the environment for symmetry with `instanceIndex`, but that doesn't make a lot of sense and duplicates what we can do via `spring.cloud.stream.instanceIndex` already.